test/integration/match_by_image_test.go

```diff
@@ -55,9 +55,9 @@ func addAlpineMatches(t *testing.T, theSource source.Source, catalog *syftPkg.Co
 		Package:       thePkg,
 		Details: []match.Detail{
 			{
-				// note: the input pURL has an upstream reference (redundant)
-				Type: "exact-indirect-match",
-				SearchedBy: map[string]any{
+				Type:       match.ExactDirectMatch,
+				Confidence: 1.0,
+				SearchedBy: map[string]interface{}{
```

Comment:
nit: why `any` -> `interface{}`?

Reply:
probably autocomplete, I'll adjust that back

Reply:
I made the rest of the file also consistent